src/pages/guardian-directory.tsx

```diff
@@ -1,12 +1,15 @@
 import { useEffect, useState } from 'react'
 import { useFlags } from 'launchdarkly-react-client-sdk'
 import { useRouter } from 'next/router'
+import { Button, Search } from '@trussworks/react-uswds'
```

Comment:
I couldn't find a reason to _not_ use the standard react-uswds search component, so that's what I did. Please let me know if you spot any problems with this approach!

Reply:
Good catch. I believe we did not use the react-uswds component because we needed a prop that was not available at the time we implemented search. But there have been a lot of updates to that component since then so I'm glad we can use it here.

Reply:
I believe I found another reason that we didn't use the component before. There is an a11y violation that is deeply nested within `<Search />` and it is causing a unit test to fail. Might have to rollback this change, or perhaps skip the test until we can address it in the ReactUSWDS repo.